src/main/java/edu/osu/waiting4ubackend/client/PetDBClient.java

```diff
@@ -160,5 +161,28 @@ public Pet updatePet(Pet pet, long petId) {
         return DBClientHelper.convertToList(petEntity.getList("status"));
     }
 
+    public List<GetUpdatesResponse> getLatestThreeUpdates() {
+        Query<Entity> query = Query.newEntityQueryBuilder()
+                .setKind(PETS_COLLECTION_NAME)
+                .setOrderBy(StructuredQuery.OrderBy.desc("dateUpdated")).build();
```

Comment:
When will the "dateUpdated" attributed be updated? Will "Edit a pet" modify the "dateUpdated"? Or it will be modified only when pet status is updated?

Reply:
Yes, this 'dateUpdated' is only focused on status updating. It won't be updated when we try to edit the pet profile.